docs/Getting Started/Arch Linux/Arch Linux Root on ZFS.rst

```diff
@@ -313,7 +313,7 @@ System Installation
         -O normalization=formD \
         -O relatime=on \
         -O xattr=sa \
-        -O mountpoint=/boot \
+        -O mountpoint=none \
```

Comment:
This should remain `mountpoint=/boot`. It's less important than the root pool having a mountpoint of `/`, but it's there for fundamentally the same reason (custom datasets).

Reply:
Reverted to `mountpoint=/boot`